### PR TITLE
feat: add support for Terraform v0.15

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,98 +7,74 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- feat: upgrade pre-commit hooks to `v0.2.3`
+- build: upgrade build-tools hooks to `v0.11.0`
 ### Fixed
-
 - Preserve case of user names added to teams in plan output
 
 ## [0.4.0]
-
 ### Added
-
```
Comment (mariux) [inline CHANGELOG.md:17]: did you change the addon? aligned style was empty line after headers...

**Review by mariux — APPROVED** [review]
lgtm

**Review by mariux — CHANGES_REQUESTED** [review]
readme not updated to document tf15 support

File: CHANGELOG.md (hunk 1 above)
Comment (soerenmartius) [reply CHANGELOG.md]: ok, aligned my vim settings to use prettier for markdown formatting

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**